Microsoft.Toolkit.Uwp.SampleApp/SamplePages/ProfileCard/ProfileCardXaml.bind

```diff
@@ -0,0 +1,18 @@
+﻿<Page xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+      xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls.Graph"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+      mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}">
+      <Grid HorizontalAlignment="Center" 
+            VerticalAlignment="Center">
+          <controls:ProfileCard 
+            x:Name="ProfileCardControl"
+            UserId="@[UserId:String: ]"
```

Comment:
Should there be something here?

Reply:
@Odonno @michael-hawker Updated as per Michael's advice